### PR TITLE
OverblogGraphiQLBundle: Added recipe for ^0.1.0

### PR DESCRIPTION
**Pull request by renatomefi**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by renatomefi** [comment]
@symfony-flex-server review again please

**Comment by renatomefi** [comment]
@symfony-flex-server review again please

File: overblog/graphiql-bundle/0.1/post-install.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<bg=blue;fg=white> Next for OverblogGraphiQLBundle </>
+    1. The bundle is auto configured now, if you need special/custom configuration read the documentation <comment>https://github.com/overblog/GraphiQLBundle/blob/master/Resources/doc/definitions/index.md</>
```
Comment (fabpot) [inline overblog/graphiql-bundle/0.1/post-install.txt]: This first comment can be removed as we don't link to the docs in post install information. What you can do instead is add the link as a comment in the configuration file (see other recipes to use the same semantic).

Comment (stof) [reply overblog/graphiql-bundle/0.1/post-install.txt]: but there is no such config file (as the default settings are meaningful).

anyway, I don't think we need to link to the doc at all from Flex here (as you only need for more advanced cases)

Comment (renatomefi) [reply overblog/graphiql-bundle/0.1/post-install.txt]: Exactly, if we put an empty file with:
`overblog_graphiql: ~`
the bot won't allow it

I've then removed the link, would still be good to point users to documentation somehow

File: overblog/graphiql-bundle/0.1/post-install.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<bg=blue;fg=white> Next for OverblogGraphiQLBundle </>
+    1. The bundle is auto configured now for <comment>dev</comment> environment only
```
Comment (fabpot) [inline overblog/graphiql-bundle/0.1/post-install.txt]: I would still remove this line which does not say anything useful.

Comment (renatomefi) [reply overblog/graphiql-bundle/0.1/post-install.txt]: Sure!

Comment (renatomefi) [reply overblog/graphiql-bundle/0.1/post-install.txt]: @fabpot done

**Pull request merged**